slave/cracker.go

```diff
@@ -0,0 +1,31 @@
+package slave
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+	"github.com/aaronang/cong-the-ripper/slave/brutedict"
+	"github.com/aaronang/cong-the-ripper/slave/hasher"
+)
+
+func Execute(task lib.Task, slave *Slave) {
+	bd := brutedict.New(&task)
+	digest, _ := b64.StdEncoding.DecodeString(task.Digest)
```

Comment:
The http request will be base64 encoded if you use []byte, that's why I suggested to keep it as []byte. The json library will automatically convert the base64 encoded string to []byte for you to populate the Job type.